src/language-js/printer-estree.js

```diff
@@ -120,6 +120,11 @@ const {
 let uid = 0;
 
 function shouldPrintComma(options, level) {
+  // angular does not allow trailing comma
+  if (options.parser === "__ng_interpolation") {
```

Comment:
I'm not sure about `__ng_action`, `__ng_binding` , `__ng_directive`, not using angular

Reply:
Trailing comma should be disabled for all of them.

Reply:
you are right, done